makefiles/boards/sam0.inc.mk

```diff
@@ -18,6 +18,11 @@ endif
 # Default for these boards is to use a CMSIS-DAP programmer
 DEBUG_ADAPTER ?= dap
 
+ifeq (dfu-util,$(PROGRAMMER))
+  DFU_USB_ID ?= ${USB_VID}:${USB_PID}
+  include $(RIOTMAKE)/tools/dfu-util.inc.mk
```

Comment:
this should not be needed anymore after @aabadie's latest changes - `$(RIOTMAKE)/tools/$(PROGRAMMER).inc.mk` is now included automatically.

Reply:
Thanks for the hint, I'll check if it works without.